packages/model-runner/src/status-codes.ts

```diff
@@ -0,0 +1,10 @@
+export enum StatusCodes {
+  Ok = 0,
+  InvalidArguments = 1,
+  UnknownError = 2,
+  DockerError = 5,
+}
+
+export function convertModelStatusCode(code: number): number {
+  return -code
```

Comment:
I assume this is done to distinguish between "the model returned a non-zero status code" and "the runner returned a non-zero status code"? We aren't running on Windows on Actions so maybe not crucial, but negative exit codes aren't always handled nicely.
Perhaps for now it's enough to just have a single non-zero status code for "the model returned a non-zero code", since we log the model exit code anyway.